backend/src/services/property.py

```diff
@@ -17,3 +17,40 @@ def add_property(self, property: Property):
     self.session.add(property)
     self.session.commit()
     return property
+
+  def get_property_by_id(self, pid) -> Property:
+      return self.db_session.query(Property).filter(Property.id == pid).one_or_none()
+
+  def get_all_property(self):
+      print("get all property")
+      return self.db_session.query(Property).all()
+
+  def delete_property(self, pid):
+      return self.db_session.query(Property).filter(Property.id == pid).delete(synchronize_session=False)
+
+  def get_properties_by_zip_code(self, zip_code):
+      return self.db_session.query(Property).filter(Property.zip_code == zip_code).all()
+
+  def get_properties_by_max_rent(self, max_rent):
+      return self.db_session.query(Property).filter(Property.monthly_rent <= max_rent).all()
+
+  def get_properties_by_bedrooms(self, bedrooms):
+      return self.db_session.query(Property).filter(Property.bedrooms == bedrooms).all()
+
+  def get_properties_by_housing_type(self, housing_types):
+      return self.db_session.query(Property).filter(Property.housing_type.in_(housing_types)).all()
+
+  def get_properties_by_date(self, date):
+      return self.db_session.query(Property).filter(Property.date_first_available <= date).all()
```

Comment:
We don't have to worry about this now since we are not actually implementing this code yet, but we will probably want to do something like this:
```suggestion
def query_properties(GetPropertyRequest):
    session.query(db.users).filter(and_(db.users.name=='Ryan', db.users.country=='England'))
```

For now, can you delete this code and add a TODO comment?

Reply:
Got it. 

I've removed these changes as Siyuan is working on the service part. I have made changes in the unit test file, property_handlers_test.py, since unit tests were failing for  get property api.